examples/pytorch/graphsage/model.py

```diff
@@ -0,0 +1,150 @@
+import torch
+import torch.nn as nn
+import dgl
+from dgl import DGLGraph
+from dgl.nn.pytorch.conv import SAGEConv
+import tqdm
+
+from dgl.multiprocessing import shared_tensor
+from dgl.utils import pin_memory_inplace, unpin_memory_inplace
+import torch.distributed as dist
```

Comment:
Logic related to these can be a bit confusing if a user is only interested in simplest usage.

Reply:
Let me add comment to clarify the last several lines are for dist/multi-gpu training